src/templates/toolkit/bet/static.html.tt

```diff
@@ -149,8 +149,8 @@
                         <div class="row-inner big-col">
                            <div class="col-inner">
                               <select id="currency" class="small_width_input"></select>
-                              <input id="amount" type="text" value="10" class="medium_width_input" pattern="^([1-9]|[1-9][0-9]|[1-9][0-9][0-9]|[1-9][0-9][0-9][0-9]|[1-9][0-9][0-9][0-9][0-9]|[1][0][0][0][0][0])$" title="100,000 maximum"/>
-                              <input id="amount_per_point" type="text" value="1" class="medium_width_input" pattern="^([1-9]|[1-9][0-9]|[1-9][0-9][0-9]|[1-9][0-9][0-9][0-9]|[1-9][0-9][0-9][0-9][0-9]|[1][0][0][0][0][0])$" title="100,000 maximum"/>
+                              <input id="amount" type="number" value="10" class="medium_width_input" min="1" max="100000"/>
```

Comment:
amount can be decimal as well, can you verify this from our current trading page